telemetry/telemetrybuffer.go

```diff
@@ -167,11 +183,13 @@ func (tb *TelemetryBuffer) BufferAndPushData(intervalms time.Duration) {
 				telemetryLogger.Printf("[Telemetry] Got data..Append it to buffer")
 				tb.payload.push(report)
 			case <-tb.cancel:
+				telemetryLogger.Printf("server cancel event")
 				goto EXIT
 			}
 		}
 	} else {
 		<-tb.cancel
+		telemetryLogger.Printf("Received cancel event")
```

Comment:
move the cancel event print statements above the EXIT tag? Or leave it if you want to know both server and client, uty